tests/acceptance/standard/test_group_labels.py

```diff
@@ -0,0 +1,125 @@
+from tests.acceptance import run_gitlabform
+
+
+class TestGroupLabels:
+    def test__can_add_a_label_to_group(self, gl, group_for_function):
+        group = gl.groups.get(group_for_function.id)
+        labels = group.labels.list()
+        assert len(labels) == 0
+
+        config_for_labels = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_labels:
+              test_label:
+                color: red
+                description: this is a label
+        """
+
+        run_gitlabform(config_for_labels, group_for_function)
+
+        updated_group = gl.groups.get(group.id)
```

Comment:
Do we need this? Can't we refer to `group`?

Reply:
@amimas - we need to get the fresh state of the group after the test run, my personal preference is to assign to a fresh parameter, I prefer to avoid mutation wherever I can to make debugging easier. We could assign it to `group` I'm easy if you would prefer to stick with that convention used elsewhere in the tests

Reply:
Ahh... Ok. Thanks for explaining. It's probably fine.

Reply:
(it comes from learned pain when I was debugging a Threaded Java app and it kept mutating variables under me in other threads 😂 )